alpaca/common/rest.py

```diff
@@ -150,6 +151,8 @@ def _one_request(self, method: str, url: str, opts: dict, retry: int) -> dict:
                 error = resp.json()
                 if "code" in error:
                     raise APIError(error, http_error)
+            else:
+                raise
```

Comment:
Why the need for an empty raise?

Reply:
oh you're right I think I wanted this to be `raise http_error` so we don't swallow any `HTTPError`s :dizzy_face: